daras_ai_v2/asr.py

```diff
@@ -889,6 +900,18 @@ def run_asr(
             ),
             # queue_prefix="gooey-gpu/short" if is_short else "gooey-gpu/long",
         )
+    elif selected_model == AsrModels.ghana_nlp_asr_v2:
+        r = requests.post(
+            f"https://translation-api.ghananlp.org/asr/v2/transcribe?language={language}",
+            headers={
+                "Content-Type": "audio/wav",
+                "Cache-Control": "no-cache",
+                "Ocp-Apim-Subscription-Key": str(settings.GHANA_NLP_SUBKEY),
+            },
+            data=requests.get(audio_url).content,
```

Comment:
forgot raise_for_status to this GET request

Reply:
also I think `audio_url_to_wav` will upload the wav file to gcp first and then downloading it here again in same func isn't efficient, have to refactor that

Reply:
@devxpy Updated the code, please review